server/routes/forms.ts

```diff
@@ -1,65 +1,91 @@
 import { type RequestHandler, Router } from 'express'
 import asyncMiddleware from '../middleware/asyncMiddleware'
 import type { Services } from '../services'
-import { FieldValidationError } from '../@types/FieldValidationError'
-import config from '../config'
-import type { AgentConfig } from '../config'
-import { components } from '../@types/strapi-api'
 import { GithubRepoRequestRequest } from '../data/strapiApiTypes'
 // import { BadRequest } from 'http-errors'
+// import { FieldValidationError } from '../@types/FieldValidationError'
 
-export default function routes({ serviceCatalogueService }: Services): Router {
+export default function routes({ serviceCatalogueService, componentNameService, dataFilterService }: Services): Router {
   const router = Router()
 
   const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
   const post = (path: string, handler: RequestHandler) => router.post(path, asyncMiddleware(handler))
 
   get('/github-repo-request-form', async (req, res) => {
-    console.log(`in get`)
-    return res.render('pages/githubRepoRequestForm')
+    const components = await componentNameService.getAllDeployedComponents()
+    const [productList] = await dataFilterService.getDropDownLists({
+      productName: '',
+      useFormattedName: true,
+    })
+
+    return res.render('pages/githubRepoRequestForm', {
+      title: 'Github Repository Requst Form',
+      productList,
+    })
+  })
+
+  // get('/products/:productName', async (req, res) => {
+  //   const { productName } = req.params
+  //   const components = await componentNameService.getAllDeployedComponentsForProduct(productName)
+  //   const [ productList ] = await dataFilterService.getDropDownLists({
+  //     teamName: '',
+  //     productName,
+  //     serviceAreaName: '',
+  //     customComponentName: '',
+  //     useFormattedName: true,
+  //   })
+
+  //   console.log(`product list`)
+
+  //   return res.render('pages/githubRepoRequestForm', {
+  //     title: 'Github Repository Requst Form',
+  //     productList,
+  //   })
+  // })
+
+  get('/products/:productName', async (req, res) => {
```

Comment:
Confused about this method? Why is it here?

Reply:
Removed it